src/components/ManageUsers/UserSearch.jsx

```diff
@@ -0,0 +1,25 @@
+const UserSearch = () => {
+  return (
+    <main>
+      <div className="usersearch__container">
+        <div>
+          <h1 className="usersearch__h1">Enter user email:</h1>
+        </div>
+        <div className="usersearch__search">
+          <input
+            className="usersearch__input"
+            type="text"
+            name="search"
+            id="search"
+            placeholder="example@lib.com"
+          />
+        </div>
+        <button type="active" className="usersearch__adduserbutton">
```

Comment:
type="active" is not a valid type. I believe you wanted to use:
```
        <button type="button" className="usersearch__adduserbutton">
```
or even better:
```
        <button type="submit" className="usersearch__adduserbutton">
```
but then you need to wrap the input and the button in the form tag. It allows submit the form by clicking enter

Reply:
Button type was changed to "button" (masło maślane)